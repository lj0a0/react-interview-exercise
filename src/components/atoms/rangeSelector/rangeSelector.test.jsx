import React from "react";
import RangeSelector from "./rangeSelector";
import renderer from "react-test-renderer";

it("RangeSelector correctly", () => {
  const tree = renderer
    .create(<RangeSelector pagesSearch={"react"} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
