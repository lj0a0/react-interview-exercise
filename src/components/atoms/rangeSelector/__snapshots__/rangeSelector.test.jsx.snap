// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RangeSelector correctly 1`] = `
<select
  className="custom-select"
  onChange={[Function]}
>
  <option
    value="6"
  >
    6
  </option>
  <option
    value="12"
  >
    12
  </option>
  <option
    value="18"
  >
    18
  </option>
  <option
    value="24"
  >
    24
  </option>
  <option
    value="30"
  >
    30
  </option>
</select>
`;
